css-three-column-layout/index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Three Column Layout</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="container">
+    <header class="row">
+      <div class="column-full">
+        <h2>Designed for life science. Optimized for SaMD and Digital Health</h2>
+      </div>
+    </header>
+
+    <section class="row">
+        <div class="column-one-third">
+          <div class="circle-background">
+            <img id="merge-image" src="images/merge.png">
+          </div>
+        </div>
+        <div class="column-one-third">
+          <div class="circle-background">
+            <img src="images/cloud-computing.png">
+          </div>
+        </div>
+        <div class="column-one-third">
+          <div class="circle-background">
+            <img src="images/discussion.png">
+          </div>
+        </div>
+    </section>
+
+    <section class="row">
+      <div class="column-one-third">
+        <h3>Streamlined QMS</h3>
+      </div>
+
+      <div class="column-one-third">
+        <h3>Integrates with existing tools</h3>
+      </div>
+
+      <div class="column-one-third">
+        <h3>In-depth Onboarding</h3>
+      </div>
+    </section>
+
+    <section class="row">
+      <div class="column-one-third">
+        <p class="textInfo">A QMS that includes modules for all stages of the product life
+          cycle from Design Control to CAPA
+        </p>
+      </div>
+      <div class="column-one-third">
+        <p class="textInfo">Enzyme QMS integrates with your existing tools, such as JIRA, GitHub and Jenkins
+          with Enzyme
+        </p>
+      </div>
+      <div class="column-one-third">
+        <p class="textInfo">We train your entire team and provide you with support when and
+          as it's needed
+        </p>
+      </div>
+    </section>
```

Comment:
I can see why you might do this, but this organization of your HTML to try to follow the Layout Classes format we discussed, but by grouping all the images, all the heading, and all the general text into their own row, your HTML looks like this content isn't related. The image, heading, and general text for each column are all related information, so they should be in a shared space in your HTML. This can be one `row` that holds three `column` elements, each of which contains the image, heading, and general text for that column.